sql/moz-fx-data-shared-prod/telemetry_derived/unified_metrics_v1/query.sql

```diff
@@ -321,7 +321,14 @@ desktop AS (
     first_seen_date,
     days_since_seen,
     os AS normalized_os,
-    normalized_os_version,
```

Comment:
Shall we have some validations of how data changes after this change? Different versions and counts. Maybe in analysis or temp.

Reply:
Thanks. Can you provide more context on how this validation would be done?  Would I create a table in a particular folder to compare counts before and after the change?

Reply:
New tables created for validations:  `analysis.um_windows_version_wchan`, `analysis.um_windows_version_wchan_1` for WHERE `submission_date = 2023-01-28`
AND `normalized_app_name = Firefox Desktop`